### ISSUE TITLE
Fail with specific exception message, stating that soffice executable from LibreOffice cannot be found

### ISSUE THREAD

**Issue by gclsoft** [benbalter/word-to-markdown#44]
```
bogon:AVOSDemo apple$ w2m /Users/apple/Downloads/Ipad.docx 
/Users/apple/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/open3.rb:211:in `spawn': no implicit conversion of nil into String (TypeError)
	from /Users/apple/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/open3.rb:211:in `popen_run'
	from /Users/apple/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/open3.rb:206:in `popen2e'
	from /Users/apple/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/2.0.0/open3.rb:372:in `capture2e'
	from /Users/apple/.rvm/gems/ruby-2.0.0-p353@global/gems/word-to-markdown-1.1.1/lib/word-to-markdown.rb:67:in `run_command'
	from /Users/apple/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/gems/2.0.0/gems/word-to-markdown-1.1.1/lib/word-to-markdown/document.rb:88:in `raw_html'
	from /Users/apple/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/gems/2.0.0/gems/word-to-markdown-1.1.1/lib/word-to-markdown/document.rb:20:in `tree'
	from /Users/apple/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/gems/2.0.0/gems/word-to-markdown-1.1.1/lib/word-to-markdown/converter.rb:79:in `semanticize_font_styles!'
	from /Users/apple/.rvm/rubies/ruby-2.0.0-p353/lib/ruby/gems/2.0.0/gems/word-to-markdown-1.1.1/lib/word-to-markdown/converter.rb:19:in `convert!'
	from /Users/apple/.rvm/gems/ruby-2.0.0-p353@global/gems/word-to-markdown-1.1.1/lib/word-to-markdown.rb:31:in `initialize'
	from /Users/apple/.rvm/gems/ruby-2.0.0-p353@global/gems/word-to-markdown-1.1.1/bin/w2m:10:in `new'
	from /Users/apple/.rvm/gems/ruby-2.0.0-p353@global/gems/word-to-markdown-1.1.1/bin/w2m:10:in `<top (required)>'
	from /Users/apple/.rvm/rubies/ruby-2.0.0-p353/bin/w2m:23:in `load'
	from /Users/apple/.rvm/rubies/ruby-2.0.0-p353/bin/w2m:23:in `<main>'
	from /Users/apple/.rvm/gems/ruby-2.0.0-p353@global/bin/ruby_executable_hooks:15:in `eval'
	from /Users/apple/.rvm/gems/ruby-2.0.0-p353@global/bin/ruby_executable_hooks:15:in `<main>'
```

**Comment by benbalter** [comment]
the Gem can't find the soffice executable. Is [LibreOffice Installed](https://github.com/benbalter/word-to-markdown#requirements-and-configuration)?.

**Comment by seanwash** [comment]
Hey @benbalter, Installing LibreOffice fixed this error for me.

Thanks!

**Comment by jesperronn** [comment]
Hi I also ran into this issue without realizing I had to install LibreOffice. Suggestion: Fail with specific exception message, stating that LibreOffice

**Comment by benbalter** [comment]
Fixed in [v 1.1.3](https://github.com/benbalter/word-to-markdown/releases/tag/v1.1.3).
